update_prevalence.py

```diff
@@ -214,15 +214,21 @@ def recent_daily_cases(self) -> List[int]:
                 # But missing data at the beginning is normal -- counties
                 # typically only show up when they have any cases.
                 self.cumulative_cases[prev] = self.cumulative_cases[current]
-            daily_cases.append(
-                self.cumulative_cases[current] - self.cumulative_cases[prev]
+            daily_cumulative_cases.append(
+                self.cumulative_cases[current]
             )
             current = prev
-        return daily_cases[::-1]
+        return daily_cumulative_cases[::-1]
 
     @property
     def cases_last_week(self) -> int:
-        return sum(self.recent_daily_cases[-7:])
+        return max(self.recent_daily_cumulative_cases[-7:]) - \
+            min(self.recent_daily_cumulative_cases[-7:])
```

Comment:
This should be `max(self.recent_daily_cumulative_cases[-8:]) - min(self.recent_daily_cumulative_cases[-8:])` - *usually* that will work out to the same as  `self.recent_daily_cumulative_cases[-1:] - self.recent_daily_cumulative_cases[-8:]`